pallets/parachain-staking/src/mock.rs

```diff
@@ -409,8 +408,9 @@ fn geneses() {
 		});
 }
 
+#[allow(unused_imports)]
```

Comment:
this is very weird compiler error, maybe because `block_author` is used only in tests it always complains that it is unused